deepdoc/parser/html_parser.py

```
@@ -105,27 +105,34 @@ def split_table(cls, html_table, chunk_token_num=512):
+            # round-trips with the original spacing, matching the Go parser
+            # (walkHTMLLeaf writes text nodes verbatim and only inserts a "\n"
+            # between block-level children, never between inline spans).
+            content = str(element)
 
-            def is_valid_html(content):
+            def is_valid_html(text):
                 try:
-                    soup = BeautifulSoup(content, "html.parser")
+                    soup = BeautifulSoup(text, "html.parser")
                     return bool(soup.find())
                 except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)